app/views/layout.erb

```diff
@@ -6,10 +6,7 @@
 </head>
 
 <body>
-  <!-- This is what a code comment in HTML looks like. -->
 
-  <!-- The `yield` below is where the contents of a view file are put. -->
-  <!-- Then the combined layout + view is sent back to the client. -->
-  <%= yield %>
```

Comment:
Deleting this code causes nothing to be shown on the page. You'll want to leave it as it was.